controller/volume_controller.go

```diff
@@ -1504,6 +1619,28 @@ func (vc *VolumeController) createReplica(v *longhorn.Volume, e *longhorn.Engine
 	return vc.ds.CreateReplica(replica)
 }
 
+func (vc *VolumeController) createReplicaStruct(v *longhorn.Volume, e *longhorn.Engine, rs map[string]*longhorn.Replica) *longhorn.Replica {
```

Comment:
will it be better to rename this to `CreateReplicaManifest`?

Reply:
good name suggestion be careful on the casing, this should be private.
ie `createReplicaManifest`

Reply:
Oh. Sorry for the typo.

Reply:
Thank you! Fixed